### ISSUE TITLE
Virtual Directories - HTTP 404 - Project Summary & Ingest Endpoints 

### ISSUE THREAD

**Issue by FinnianDempsey** [OctopusDeploy/Issues#7863]
### Team

- [X] I've assigned a team label to this issue

### Severity

Blocking customers on latest version

### Version

2022.3.10674

### Latest Version

I could reproduce the problem in the latest build

### What happened?

If Octopus Server has bindings configured to use Virtual Directories, HTTP 404 errors are returned when navigating the Projects page. This is preventing users from creating new projects. 

### Reproduction

Configure Octopus Server to use a virtual directory binding e.g. https://<octopusUrl>/octopus
Navigate to the Projects Page
See 404

### Error and Stacktrace

```shell
/bff/spaces/Spaces-1/projects/summaries:1 Failed to load resource: the server responded with a status of 404 (Not Found)

/ingest Failed to load resource: the server responded with a status of 404 (Not Found)
```

### More Information

[Internal Link - Slack](https://octopusdeploy.slack.com/archives/C02PAUNL6J0/p1666571613417459)
[Internal Link - Zendesk](https://octopus.zendesk.com/agent/tickets/99192)
[Discourse](https://help.octopus.com/t/project-page-reporting-404-and-not-loading/28496)

### Workaround

Configure Octopus to not use a Virtual Directory

**Comment by CDSGlobalUK** [comment]
This is a major issue, as it also stopped any deployments from happening

**Comment by bojake** [comment]
This occurs in 2022.3.10686 installed today. 

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://(server)/api/telemetry/process. (Reason: CORS request did not succeed). Status code: (null)

Uncaught (in promise) Error: 
    a octopusError.ts:25
    f ajax.ts:259
    handleError ajax.ts:198
    onreadystatechange ajax.ts:86

Unable to ingest messages: Error 
Object { ErrorMessage: "Error" }

GET /bff/spaces/Spaces-1/projects/summaries -> 404

GET on octopus/api/Spaces-1/projects/Projects-1/logo?cb=Projects-1 is returning status 500

**Comment by bojake** [comment]
my workaround:

ADD another binding using the Octopus Manager that used the root binding (no virtual directory as indicated in the stated workaround).

This fixes the projects and tasks page. We have two bindings now, one for "/octopus" and the other on "/" (the root).

**Comment by brentkinney** [comment]
Another report of this issue - https://octopus.zendesk.com/agent/tickets/99354

**Comment by CDSGlobalUK** [comment]
This to have been fixed in the latest build 10723

**Comment by paraicoceallaigh** [comment]
another instance of this [Internal Link]: https://octopus.zendesk.com/agent/tickets/106910
